### ISSUE TITLE
Outline is not visible on checkbox/radio button in high contrast mode

### ISSUE THREAD

**Issue by albenakertova** [vmware/clarity#5723]
When in Windows high contrast mode no outline is visible on checkboxes and radio buttons.

1.  Go to Clarity documentation -> Angular components -> Checkbox or Radio: https://clarity.design/angular-components/checkbox/
2.  Enter high contrast mode  (toggle with left Alt + left Shift + Print) 
3.  Move between examples with tab key
4.  Observe that no outline is visible around checkboxes and radio buttons in high contrast.

## Expected behavior

An outline should be visible. This is working correct for Clarity Core checkbox/radio.

Note: A lot of our clr-datagrids use the checkbox provided by the Angular component and these cannot be  implemented with the new Core components.

## Versions
v.5

**Clarity project:**

- [ ] Clarity Angular/UI

**Framework version:**
_ie: Angular 11_

**Device:**

- OS: Windows
- Browser: Firefox

**Comment by bdryanovski** [comment]
@albenakertova thanks for reporting this.

**Comment by Jinnie** [comment]
Fixed in v12 and v5
